### ISSUE TITLE
IDA Plugin should not use PE format in .ini until supported

### ISSUE THREAD

**Issue by cryzlasm** [airbus-seclab/bincat#51]
```
Env: WIN7 X64  IDA7 
        x86_64-7.2.0-release-win32-seh-rt_v5-rev1_mingv64_GCC

Please Tell Me, How To Fix It?
Thank you.

==============================================


analyzer.log
------------------------------------------------------------------
[INFO]  main: BinCAT version v0.7
[EXCEPTION] parser: failed to load header 
Sys_error(": No such file or directory")
Raised by primitive operation at file "pervasives.ml", line 366, characters 28-54
Called from file "pervasives.ml" (inlined), line 374, characters 2-47
Called from file "c2newspeak/typedC.ml", line 345, characters 14-30
Called from file "frontend/parser.mly", line 135, characters 15-33
[ABORT] main: PE file format not implemented yet
Raised by primitive operation at file "utils/log.ml", line 157, characters 41-69
Called from file "main.ml", line 70, characters 19-76
Called from file "bincat.ml", line 25, characters 4-55
[EXCEPTION] main: Exception caught in main loop
Exceptions.Error("PE file format not implemented yet")
Raised at file "utils/log.ml", line 160, characters 4-32
Called from file "main.ml", line 70, characters 19-76
[STOP] nothing analyzed
------------------------------------------------------------------

IDAPythonLog
------------------------------------------------------------------
INFO:bincat.gui:Launching the analyzer
DEBUG:netnode:set: data length: 0xf86
DEBUG:netnode:set: chunk run: 0x1 to 0x3 (0x3 chunks)
DEBUG:netnode:set: data length: 0x22
DEBUG:netnode:set: data length: 0xf86
DEBUG:netnode:set: chunk run: 0x1 to 0x3 (0x3 chunks)
DEBUG:netnode:set: data length: 0x22
DEBUG:bincat.plugin:Using C:\Users\Administrator\Desktop\Dbgview.exe as source binary path
DEBUG:bincat.plugin:Current analyzer path: c:\users\admini~1\appdata\local\temp\tmpcvjx2ibincat
DEBUG:bincat.plugin:Generating .no files...
DEBUG:bincat.plugin:Initial header files: ['']
DEBUG:bincat.plugin.npkgen:Generating TNPK file in c:\users\admini~1\appdata\local\temp\tmpr2zzrcbincat-generate-header
ERROR:bincat.plugin.npkgen:Error encountered while running c2newspeak.
--- start of c2newspeak output ---
Fatal error: c:\\users\\admini~1\\appdata\\local\\temp\\tmpr2zzrcbincat-generate-header\\ida-generated.h:233#0: directive #pragma pack(push, 1) not supported yet, rewrite your code or try option --ignore-pragma

--- end of c2newspeak output ---
Traceback (most recent call last):
  File "C:/Users/Administrator/AppData/Roaming/Hex-Rays/IDA Pro/plugins\idabincat\npkgen.py", line 130, in generate_tnpk
    "pre-processed.c"], stderr=subprocess.STDOUT)
  File "C:\python27-x64\Lib\subprocess.py", line 219, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
CalledProcessError: Command '['c2newspeak', '--typed-npk', '-o', 'c:\\users\\admini~1\\appdata\\local\\temp\\tmpr2zzrcbincat-generate-header\\pre-processed.no', 'pre-processed.c']' returned non-zero exit status 1
WARNING:bincat.plugin:.no file containing type data for the file being analyzed could not be generated, continuing. The ida-generated header could be invalid.
DEBUG:bincat.plugin:Final npk files: []
DEBUG:bincat.plugin:Analyzer cmdline: [bincat c:\users\admini~1\appdata\local\temp\tmpcvjx2ibincat\init.ini c:\users\admini~1\appdata\local\temp\tmpcvjx2ibincat\out.ini c:\users\admini~1\appdata\local\temp\tmpcvjx2ibincat\analyzer.log]
DEBUG:bincat.plugin:Analyzer new state: Starting
DEBUG:bincat.plugin:Analyzer new state: Running
INFO:bincat.plugin:Analyzer: starting process
INFO:bincat.plugin:Analyzer started.
DEBUG:bincat.plugin:Analyzer new state: Not running
INFO:bincat.plugin:Analyzer process finished
ERROR:bincat.plugin:analyzer returned exit code=2
INFO:bincat.plugin:---- stdout ----------------
INFO:bincat.plugin:BinCAT v0.7

INFO:bincat.plugin:---- stderr ----------------
INFO:bincat.plugin:EXCEPTION: Exceptions.Error("PE file format not implemented yet")
Check log file for details [c:\users\admini~1\appdata\local\temp\tmpcvjx2ibincat\analyzer.log]

DEBUG:bincat.plugin:---- logfile ---------------
DEBUG:bincat.plugin:[INFO]  main: BinCAT version v0.7
DEBUG:bincat.plugin:[EXCEPTION] parser: failed to load header
DEBUG:bincat.plugin:Sys_error(": No such file or directory")
DEBUG:bincat.plugin:Raised by primitive operation at file "pervasives.ml", line 366, characters 28-54
DEBUG:bincat.plugin:Called from file "pervasives.ml" (inlined), line 374, characters 2-47
DEBUG:bincat.plugin:Called from file "c2newspeak/typedC.ml", line 345, characters 14-30
DEBUG:bincat.plugin:Called from file "frontend/parser.mly", line 135, characters 15-33
DEBUG:bincat.plugin:[ABORT] main: PE file format not implemented yet
DEBUG:bincat.plugin:Raised by primitive operation at file "utils/log.ml", line 157, characters 41-69
DEBUG:bincat.plugin:Called from file "main.ml", line 70, characters 19-76
DEBUG:bincat.plugin:Called from file "bincat.ml", line 25, characters 4-55
DEBUG:bincat.plugin:[EXCEPTION] main: Exception caught in main loop
DEBUG:bincat.plugin:Exceptions.Error("PE file format not implemented yet")
DEBUG:bincat.plugin:Raised at file "utils/log.ml", line 160, characters 4-32
DEBUG:bincat.plugin:Called from file "main.ml", line 70, characters 19-76
DEBUG:bincat.plugin:[STOP] nothing analyzed
DEBUG:bincat.plugin:====== end of logfile ======
DEBUG:bincat.plugin:Parsing analyzer result file
ERROR:bincat.plugin:Could not parse result file
------------------------------------------------------------------
```

**Comment by trou** [comment]
Sorry it's because the interface is misleading.
The log says:
`DEBUG:bincat.plugin:[ABORT] main: PE file format not implemented yet`

You should use "remap" binary when analyzing PE files for now.

**Comment by cryzlasm** [comment]
Thank you. I'll give it a try

**Comment by trou** [comment]
Fixed in 3e1ced643daeb2a6677c34a2d59073c65f996f19
